share/completions/journalctl.fish

```diff
@@ -34,7 +34,7 @@ complete -c journalctl -f -s a -l all -d 'Show all fields in full'
 complete -c journalctl -f -s f -l follow -d 'Show live tail of entries'
 complete -c journalctl -f -s n -l lines -d 'Controls the number of journal lines'
 complete -c journalctl -f -l no-tail -d 'Show all lines, even in follow mode'
-complete -c journalctl -f -s o -l output -d 'Controls the formatting' -xa 'short short-monotonic verbose export json json-pretty json-sse cat'
+complete -c journalctl -f -s o -l output -d 'Controls the formatting' -xa '(journalctl --output=help | while read style; echo -e "$style\t"; end)'
```

Comment:
You can do `(printf %s\t\n (journalctl --output=help))`.

Using `read` like this sets a global variable by default, or would even update a universal `style` variable if such exists.
This can be remedied by using `read --local`.

Reply:
Fantastic, thank you!